### PR TITLE
Set the value of the CodeMirror on first load

### PR DESCRIPTION
**Pull request by santiweight**
Hey there. There is an issue with the current code that means that sometimes the CodeMirror text doesn't show at start, due to the lack of setValue on initiation. The other issue is that refreshes weren't occurring after setValue.

### CODE REVIEW

File: src/Reflex/CodeMirror/FFI.hs [hunk 1]
```diff
@@ -60,7 +60,7 @@ setValueAndRefresh ref text = do
         refreshMirror :: Object -> JSM ()
         refreshMirror codemirror = do
             let refreshFun = fun (\_ _ _ -> void $ codemirror ^. js0 "refresh")
-            _ <- jsg2 "setTimeout" refreshFun (1 :: Double)
+            _ <- jsg2 "setTimeout" refreshFun (100 :: Double)
```
Comment (srid) [inline src/Reflex/CodeMirror/FFI.hs:63]: Does this work reliably? Why the change to timeout value?

Comment (santiweight) [reply src/Reflex/CodeMirror/FFI.hs:63]: Unfortunately I think this is an imprecise science... When I used a value of 1ms, the results were non-deterministic. With 100ms (in my larger reflex application), the codemirror always showed the text after around 40 tries (as opposed to 50% with 1ms before). There's no way, afaik to have this be more precise.

**Comment by srid** [comment]
@santiweight I don't think this change is necessary, especialy given the timeout-based reliability issue.

It can be solved by using the codemirror [autorefresh plugin](https://codemirror.net/doc/manual.html#addon_autorefresh), which also polls in case the first delay didn't work out.

Include that plugin JS, and set `configuration_autoRefresh` of your codemirror config to `True`. 

This is what I do in [Cerveau](https://www.cerveau.app/) and it works.
